manifests/server/export.pp

```diff
@@ -7,7 +7,7 @@
 # Parameters
 #
 # @param clients
-#   String or Array. Sets the allowed clients and options for the export in the exports file.
+#   String. Sets the allowed clients and options for the export in the exports file.
 #   Defaults to <tt>localhost(ro)</tt>
```

Comment:
puppet-strings already generates the data type and default value automatically, so we can trim this down even more:

```suggestion
#   Sets the allowed clients and options for the export in the exports file.
```

Reply:
@Safranil sounds good.